api/config/swagger.py

```diff
@@ -705,4 +724,4 @@ def schema_view_swagger(request):
 
         }
     )
-    return response.Response(schema)
+    return response.Response(schema)
```

Comment:
@a-braham Could use autopep8  to make sure a new line is added at the end of the file and everything is to standard